docs/captures/managed-storage-account.md

```diff
@@ -0,0 +1,130 @@
+# Managed Storage Account
+
+## Motivation
+
+Retina Capture helps customers capture network packets on Kuberentes cluster to debug network issues. Before Retina Capture can debug the packets, Retina Capture can store the network packets and the customers need to download the packets from Retina Capture supported locations with tools like Wireshark.
+
+To simplify customers' work to decide where to store the packets and then download the packets to local environment, we propose a managed account solution for customer to help customer manage the lifecycle, security when to the storage account and garbage collection of the storage account. Customers can use managed storage account by default, or specify their own storage account ID and Storage Blob Data Owner or higher privilege should be granted to [AKS managed identity or service principal](https://learn.microsoft.com/en-us/azure/aks/use-managed-identity).
+
+## Detailed Design
+
+### Workflow
+
+![Workflow of Retina Capture without managed storage account](img/capture-managed-storage-account.png "Workflow of Retina Capture without managed storage account")
+
+## Setup
+
+To enable the managed storage account, you need to specify the following configuration in the helm command,
+
+```bash
+--set operator.enabled=true \
+--set operator.capture.enableManagedStorageAccount=true
+```
+
+Internally, `enableManagedStorageAccount` will change the following retina-operator configuration and azure credential config as explained in the following two sections.
+
+### Retina-Operator Configuration
+
+#### enableManagedStorageAccount Configuration
+
+`enableManagedStorageAccount` configuration controls whether to use managed storage account or not.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: retina-operator-config
+  namespace: kube-system
+data:
+  ... ...
+  enableManagedStorageAccount: true/false
+  azureCredentialConfig: /etc/kubernetes/cloud-config/azure.json
+```
+
+#### Azure credential configuration
+
+The configuration `azureCredentialConfig` in retina-operator-config indicates the Azure credential file path to talk to azure for the managed resources.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: retina-operator-config
+  namespace: kube-system
+data:
+  .. ...
+  enableManagedStorageAccount: true/false
+  azureCredentialConfig: /etc/kubernetes/cloud-config/azure.json
+```
+
+The volume defined in retina-operator deployment mount the secret containing the authentication information to the container.
+
+```yaml
```

Comment:
nit: This is the yaml is the same with the one above, why don't you only keep one to show how to use both enabling and credentials.